js/index.js

```diff
@@ -40,3 +40,91 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+// nav
+
+const anchors = document.querySelectorAll('a');
+// anchors[0].textContent = siteContent['nav']['nav-item-1'];
+// anchors[1].textContent = siteContent['nav']['nav-item-2'];
+// anchors[2].textContent = siteContent['nav']['nav-item-3'];
+// anchors[3].textContent = siteContent['nav']['nav-item-4'];
+// anchors[4].textContent = siteContent['nav']['nav-item-5'];
+// anchors[5].textContent = siteContent['nav']['nav-item-6'];
+
+anchors.forEach((item, num) => item.textContent = siteContent['nav'][`nav-item-${num+1}`]);
+
+
+// Changing nav text to green
+
+anchors.forEach(item => item.style.color = 'green');
+
+
+// Adding two new items to navigation system
+
+let navBar = document.querySelector('nav');
+
+const newElement1 = document.createElement('a');
+newElement1.href = '#';
+newElement1.textContent = 'Blog';
+newElement1.style.color = 'green';
+navBar.prepend(newElement1);
+
+const newElement2 = document.createElement('a');
+newElement2.href = '#';
```

Comment:
You did a great job appending and prepending DOM elements to the navBar! A lot of people stumble on this one. I also like that you made them green, because a lot of people just left them gray. I'd just like to correct they way you're trying to set the href here. You actually have to use setAttribute, because href is an attribute of the 'a' tag. 

https://www.w3schools.com/jsref/met_element_setattribute.asp

```suggestion
newElement2.setAttribute('href' , '#');
```